multiversx_sdk_cli/interfaces.py

```diff
@@ -39,9 +39,12 @@ def get_data(self) -> str:
         return ""
 
 
-class IAccount:
+class IAccount(Protocol):
     def sign_transaction(self, transaction: ITransaction) -> str:
-        return ""
+        ...
+    
+    def serialize_for_signing(self) -> bytes:
```

Comment:
All right for now. An alternative could have been to create a `Message` such as `message = Message(account.pubkey)` and pass that to the `ValidatorSigner`. `Message` is a class from multiversx-sdk-core.